invenio_files_rest/config.py

```diff
@@ -100,3 +107,9 @@
 
 FILES_REST_MULTIPART_EXPIRES = timedelta(days=4)
 """Time delta after which a multipart upload is considered expired."""
+
+FILES_REST_TASK_WAIT_INTERVAL = 2.0
```

Comment:
timedelta vs seconds: because of the time scales they are operating on (days, months, years) for multipart expiry vs seconds for the task waiting.